packages/react/src/patterns/blocks/ContentBlockMedia/ContentBlockMediaGroup.js

```diff
@@ -26,30 +26,30 @@ const { prefix } = settings;
  * @param {Array} props.link use case link object
  * @returns {object} JSX Object
  */
-const UseCasesGroup = ({ usecaseGroup: { title, image, lists, link } }) => {
+const ContentBlockMediaGroup = ({
```

Comment:
This might actually be what is supposed to be the `ContentGroupSimple`. I think @RaphaelAmadeu is working on that now.

Reply:
Cool, will work it out with @RaphaelAmadeu and see where he is with it and how to incorporate. Might need to be a separate story for another sprint if he's currently working on it.